hack/lib/cleanup.sh

```diff
@@ -152,4 +152,18 @@ function os::cleanup::tmpdir() {
 		${USE_SUDO:+sudo} rm -rf "${directory}"
 	done
 }
-readonly -f os::cleanup::tmpdir
\ No newline at end of file
+readonly -f os::cleanup::tmpdir
+
+# os::cleanup::dump_events dumps all the events from a cluster to a file.
+#
+# Globals:
+#  ARTIFACT_DIR
+# Arguments:
+#  None
+# Returns:
+#  None
+function os::cleanup::dump_events() {
+	os::log::info "Dumping cluster events to ${ARTIFACT_DIR}/events.txt"
```

Comment:
Not sure if it's worth moving at "debug". More often than not when debugging e2e I would prefer to see by default that a file with all the events has been placed somewhere. It's the same level as the dump we get from etcd, too.